src/fidesops/core/config.py

```diff
@@ -39,8 +39,6 @@ class ExecutionSettings(FidesSettings):
     TASK_RETRY_BACKOFF: int
     REQUIRE_MANUAL_REQUEST_APPROVAL: bool = False
     MASKING_STRICT: bool = True
-    CELERY_BROKER_URL: Optional[str] = None
-    CELERY_RESULT_BACKEND: Optional[str] = None
     WORKER_ENABLED: bool = True
```

Comment:
I've intentionally left `WORKER_ENABLED` in here, as whether the implementation is deploying a separate worker, and whether it's overriding the Celery config are two separate concerns.